### PR TITLE
Add check to override user lang choose if LOCALES is disabled

### PR DESCRIPTION
**Pull request by wdhda-ch**
Refactored, add check and tests

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/utils/localise.ts [hunk 1]
```diff
@@ -50,3 +48,11 @@ export function getLocalesField(fieldName: string, req: Request): string {
     }
 
 }
+
+export function getLanguageChoice(req: Request): string {
+    const QUERY_LANG = "lang";
```
Comment (amartin7663) [inline src/utils/localise.ts:53]: Do we need this as a separate variable, could we not just pass the string straight to the function

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by amartin7663 — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by amartin7663 — APPROVED** [review]

**Pull request merged**